packages/expo-updates/cli/utils/args.ts

```diff
@@ -40,3 +40,14 @@ export function assertArgs(schema: arg.Spec, argv: string[]): arg.Result<arg.Spe
     throw error;
   }
 }
+
+export function assertArg(args: arg.Result<arg.Spec>, name: any, type: 'string' | 'number'): any {
+  const value = args[name];
+  if (value === undefined || value === null) {
+    Log.exit(`${name} must not be null`);
```

Comment:
```suggestion
    Log.exit(`${name} must be provided`, 1);
```

(null is more of an implementation detail)